subprojects/distributions/src/changes/accepted-public-api-changes.json

```diff
@@ -91,6 +91,534 @@
+                "org.gradle.api.DefaultTask.newOutputDirectory()",
+                "org.gradle.api.DefaultTask.newOutputFile()"
+            ]
+        },
+        {
+            "type": "org.gradle.api.reporting.GenerateBuildDashboard",
+            "member": "Class org.gradle.api.reporting.GenerateBuildDashboard",
+            "acceptation": "Methods removed",
+            "changes": [
+                "org.gradle.api.DefaultTask.newInputDirectory()",
+                "org.gradle.api.DefaultTask.newInputFile()",
+                "org.gradle.api.DefaultTask.newOutputDirectory()",
+                "org.gradle.api.DefaultTask.newOutputFile()"
+            ]
+        },
+        {
+            "type": "org.gradle.api.reporting.model.ModelReport",
+            "member": "Class org.gradle.api.reporting.model.ModelReport",
+            "acceptation": "Methods removed",
+            "changes": [
+                "org.gradle.api.DefaultTask.newInputDirectory()",
+                "org.gradle.api.DefaultTask.newInputFile()",
+                "org.gradle.api.DefaultTask.newOutputDirectory()",
+                "org.gradle.api.DefaultTask.newOutputFile()"
+            ]
+        },
+        {
+            "type": "org.gradle.api.tasks.AbstractCopyTask",
+            "member": "Class org.gradle.api.tasks.AbstractCopyTask",
+            "acceptation": "Methods removed",
+            "changes": [
+                "org.gradle.api.DefaultTask.newInputDirectory()",
+                "org.gradle.api.DefaultTask.newInputFile()",
+                "org.gradle.api.DefaultTask.newOutputDirectory()",
+                "org.gradle.api.DefaultTask.newOutputFile()"
+            ]
+        },
+        {
+            "type": "org.gradle.api.tasks.AbstractExecTask",
+            "member": "Class org.gradle.api.tasks.AbstractExecTask",
+            "acceptation": "Methods removed",
+            "changes": [
+                "org.gradle.api.DefaultTask.newInputDirectory()",
+                "org.gradle.api.DefaultTask.newInputFile()",
+                "org.gradle.api.DefaultTask.newOutputDirectory()",
+                "org.gradle.api.DefaultTask.newOutputFile()"
+            ]
+        },
+        {
+            "type": "org.gradle.api.tasks.ant.AntTarget",
+            "member": "Class org.gradle.api.tasks.ant.AntTarget",
+            "acceptation": "Methods removed",
+            "changes": [
+                "org.gradle.api.DefaultTask.newInputDirectory()",
+                "org.gradle.api.DefaultTask.newInputFile()",
+                "org.gradle.api.DefaultTask.newOutputDirectory()",
+                "org.gradle.api.DefaultTask.newOutputFile()"
+            ]
+        },
+        {
+            "type": "org.gradle.api.tasks.Delete",
+            "member": "Class org.gradle.api.tasks.Delete",
+            "acceptation": "Methods removed",
+            "changes": [
+                "org.gradle.api.DefaultTask.newInputDirectory()",
```

Comment:
I should be enough to declare the removal on `DefaultTask`. Doesn't this work?

Reply:
I tried that, I got this:

```text
> The following regressions are declared as accepted, but didn't match any rule:

 org.gradle.api.DefaultTask - Class org.gradle.api.DefaultTask - [org.gradle.api.DefaultTask.newInputDirectory(), org.gradle.api.DefaultTask.newInputFile(), org.gradle.api.DefaultTask.newOutputDirectory(), org.gradle.api.DefaultTask.newOutputFile()]
```